packages/bigquery/index.cjs

```diff
@@ -1,5 +1,29 @@
 const { BigQuery } = require('@google-cloud/bigquery');
-const { EvidenceType, TypeFidelity } = require('@evidence-dev/db-commons');
+const { EvidenceType, TypeFidelity, getEnv } = require('@evidence-dev/db-commons');
+
+const envMap = {
+	projectId: [
+		{ key: 'EVIDENCE_BIGQUERY_PROJECT_ID', deprecated: false },
+		{ key: 'BIGQUERY_PROJECT_ID', deprecated: false },
+		{ key: 'project_id', deprecated: true },
+		{ key: 'PROJECT_ID', deprecated: true },
+
+	],
+	credentials: {
+		client_email: [
+			{ key: 'EVIDENCE_BIGQUERY_CLIENT_EMAIL', deprecated: false },
+			{ key: 'BIGQUERY_CLIENT_EMAIL', deprecated: false },
+			{ key: 'client_email', deprecated: true },
+			{ key: 'CLIENT_EMAIL', deprecated: true }
+		],
+		private_key: [
+			{ key: 'EVIDENCE_BIGQUERY_PRIVATE_KEY', deprecated: false },
+			{ key: 'BIGQUERY_PRIVATE_KEY', deprecated: false },
+			{ key: 'private_key', deprecated: true },
+			{ key: 'PRIVATE_KEY', deprecated: true }
+		]
+	}
+};
 
```

Comment:
RE: `private_key`, `client_email` etc.  
It seems to me that we are using a mix of camel or snake for our var names. Should we settle on one?  (Not specific to this change).

Reply:
Ah yeah I hadn't noticed this; should probably be updated to be camel in this case.